rs/execution_environment/src/scheduler.rs

```diff
@@ -654,6 +655,7 @@ impl SchedulerImpl {
 
         let mut heartbeat_and_timer_canister_ids = BTreeSet::new();
         let mut round_executed_canister_ids = BTreeSet::new();
+        let mut round_fully_executed_canister_ids = BTreeSet::new();
```

Comment:
NIT: I'd add a clarifying comment that those are canister IDs for the canisters that had a full message execution. because seeing 2 similar names (`round_executed_canister_ids` & `round_fully_executed_canister_ids`) next to each other can bring some confusion.